### PR TITLE
Squash bug/DLSM-332-make-filecloud-php-fit-for-php7

### PR DESCRIPTION
**Pull request by PStiemer**
Background:
A PHP7 compatibility check revealed a mistake in the generated classes.
The API request "2falogin" is not a valid function name, since it begins with a number.

Since i had to regenerate the code i updated to the current api version.
I also wrote updateOperationIdMappingFile.php which adds new api operations to the Mapping file.

Afterwards i updated the readme.

That means that the following files need to be reviewed:
- apidoc2swagger/README.md
- apidoc2swagger/operationidmapping.txt
- apidoc2swagger/updateOperationIdMappingFile.php

#DLSM-332 work 4h

### CODE REVIEW

File: apidoc2swagger/operationidmapping.txt [hunk 1]
```diff
@@ -117,3 +117,30 @@ app/searchext/=searchExt
...
+/core/getuserconsent=getUserConsent
+/core/privacyconsentaccept=privacyConsentAccept
+/core/getprofileimage=getProfileImage
+/core/getmostactiveshares=getMostActiveShares
+/core/getrecentshareactivityforuser=getRecentShareActivityForUser
+/core/getshareactivityforshare=getShareActivityForShare
+/core/addshortcut=addShortcut
+/core/deleteshortcut=deleteShortcut
+/core/getshortcuts=getShortcuts
+/core/getoourl=getOOUrl
```
Comment (Shyru) [inline apidoc2swagger/operationidmapping.txt:145]: Is that an o or zero in the name? I would guess that the url contains o and as such we should also use O (capital o) in our method name?

File: apidoc2swagger/operationidmapping.txt [hunk 2]
```diff
@@ -117,3 +117,30 @@ app/searchext/=searchExt
...
+/core/verifyoourl=verifyOOUrl
```
Comment (Shyru) [inline apidoc2swagger/operationidmapping.txt:146]: Same here.

**Review by Shyru — CHANGES_REQUESTED** [review]
Good work, also adding the helper tool to update the mapping file more easily.
But please also update the toplevel README.md. (Especially the FileCloud API Version and the required PHP-Version (i think it is 5.5 now, see composer.json))

File: apidoc2swagger/operationidmapping.txt (hunk 1 above)
Comment (PStiemer) [reply apidoc2swagger/operationidmapping.txt:145]: These are capital o's

**Comment by PStiemer** [comment]
The toplevel readme.md is already up to date.

image

**Review by Shyru — APPROVED** [review]

**Pull request merged**